server/controllers/DocumentControllers.js

```diff
@@ -0,0 +1,215 @@
+import pagination from '../helper/pagination';
+
+const Document = require('../models').Document;
+const User = require('../models').User;
+
+const DocumentControllers = {
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns a response object.
+   */
+  create(req, res) {
+    const authorId = res.locals.decoded.UserId;
+    return Document.create({
+      title: req.body.title,
+      content: req.body.content,
+      UserId: authorId,
+      access: req.body.access
+    })
+    .then(document => res.status(201).send({
+      message: 'Document successfully created',
+      document,
+    })
+    )
+    .catch(() => res.status(400).send({
+      message: 'Please verify your input'
+    }));
+  },
+
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns the list of documents available
+   */
+
+  listDocuments(req, res) {
```

Comment:
You can make your endpoints RESTful such that the methods in the DocumentsController would be `create, index(i.e to list all documents), show(to get a single document), update and destroy`